proposals/lip-0058.md

```diff
@@ -401,7 +445,7 @@ The BFT parameters valid at the height `h`.
 ##### Execution
 
 ```python
-getBFTParametersInternal(h):
+def getBFTParametersInternal(h):
```

Comment:
Here and later, I would still add the typing

Reply:
I would prefer not to do it as I wanted to restrict the PR to necessary specification changes, we have all the "Parameters" section which already describe the input parameters, it would not help to save additional validations and there are also a lot of complex objects as input parameters. I think we also said that introducing type hinting for published LIPs is optional.

Reply:
sounds good.